src/Codec/Xlsx/Writer/Stream.hs

```diff
@@ -176,11 +185,12 @@ writeXlsxWithSharedStrings settings sstable items = do
   pure res
 
 -- massive amount of boilerplate needed for excel to function
-boilerplate :: forall m . PrimMonad m  => Map Text Int -> [(ZipEntry,  ZipData m)]
-boilerplate sstable =
+boilerplate :: forall m . PrimMonad m  => WriteSettings -> Map Text Int -> [(ZipEntry,  ZipData m)]
+boilerplate settings sstable =
```

Comment:
What does `sstable` refer to here? Again, I'm not familiar with the spec so perhaps it's obvious to someone with the appropriate domain knowledge. Is it an "SS Table"? If so, what does "SS" refer to?

Reply:
shared strings table, it's a pretty big thing, the public API has documentation for it.

Reply:
Ah, yes. Thanks for clarifying. Do you think perhaps we ought to rename that binding to, _e.g._ `sharedStrings`? Normally I think terse names are fine given that their meaning is often clear from the type signature, but I don't think it's so clear to the reader in this case.

Reply:
sure, but it was already bound to the public api function, so I called it `sharedStrings'`